common/ManagementTestShared/MgmtRecordTestBase.cs

```diff
@@ -0,0 +1,202 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+
+using Azure.Core;
+using Azure.Core.TestFramework;
+using Azure.Management.Resources;
+using System;
+
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using NUnit.Framework;
+
+
+namespace Azure.ResourceManager.TestFramework
+{
+    [Category("Recorded")]
+    public abstract class MgmtRecordTestBase<TEnvironment> : ClientTestBase where TEnvironment : TestEnvironment, new()
```

Comment:
I don't think you should copy the implementation of RecordedTestBase. Any reason not to inherit from it?